packages/lesswrong/components/ea-forum/EASurveyBanner.tsx

```diff
@@ -94,23 +95,24 @@ const EASurveyBanner = ({classes}: {classes: ClassesType}) => {
     captureEvent("ea_forum_survey_closed");
   }, [hideBanner, captureEvent]);
 
-  if (cookies[cookieName] === "true") {
+  if (cookies[HIDE_EA_FORUM_SURVEY_BANNER_COOKIE] === "true") {
     return null;
   }
 
   const {ForumIcon} = Components;
   return (
     <DeferRender ssr={!!currentUser}>
       <div className={classes.root}>
-        Take the 4 minute EA Forum Survey to help inform our strategy and funding decisions
-        <TypeformPopupEmbed
-          widgetId="Z1wH4v8v"
-          domain="cea-core.typeform.com"
-          title="EA Forum survey"
-          label="Take the survey"
-          onSubmit={onSubmitSurvey}
+        Take the 2024 EA Forum Survey to help inform our strategy and priorities
+        <Link
+          to="https://forms.cea.community/test?utm_source=ea_forum&utm_medium=banner"
```

Comment:
TODO update this link once we've set up the real survey (this links to the test version which should be fully functional)